tests/codestyle/fixtures/bracesFixer/expected.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
as I understand it, this PR deals with fixing braces formatting around `{` and parentheses `(`.
This expected/actual pair seems to suggest that we're also adding `declare(strict_types=1);` and changing `string` into `?string`